VarshaSaroj/Task2.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+
+<head>
+    <title>Retro City</title>
+    <style>
+		div {
+		color: black;
+        position:absolute;
+        bottom: 1px;
+        right: 25px;
+        width: 250px;
+        height: 80px;
+        padding:1px;
+		padding-top:5px;
+		padding-bottom:10px;
+        padding-left:5px;
+        font-size:20px;
+        background-color:#e97e37f8;
+        box-shadow: 10px 10px #ae83b3f8;
+	
+      }
+    </style>
+</head>
+<body>
+    <img src="C:\Users\Hp\Downloads\2734813.jpg" height="100%" width="100%" alt="This is a image">
```

Comment:
image url is wrong here, change it to `../assets/images/task2.jpg` or `https://github.com/CC-BHU/web-development/raw/main/assets/images/task2.jpg`.

Reply:
But I downloaded it from web

Reply:
The image is present in your local system, and if I view the webpage I won't be able to see that.
I wont be able to locate that file because its present in your computer.

If you see a image is provided in assets directory, either you link to that or use image directly from url

`../assets/images/task2.jpg` OR `https://github.com/CC-BHU/web-development/raw/main/assets/images/task2.jpg`